### DISCUSSION TITLE
Connecting to remote jupyter hub notebooks

### DISCUSSION THREAD

**Discussion by crazy4pi314** [microsoft/vscode-jupyter#11570] [Questions and Answers]
I am looking to be able to view/edit remote notebooks on when I connect to a remote kernel (i.e. Jupyter Hub hosted in an Azure VM). I seem to be able to connect a local notebook to that kernel, but maybe I am missing a/the way to edit the remote files. Thanks for the help!

**Comment by DonJayamanne** [comment]
there's an issue tracking this feature request, will find that and paste that here
